projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -1391,23 +1509,111 @@ public function render_outline_label( $id, $label, $required, $required_field_te
+		$outline_classes = $this->get_attribute( 'outlinestyleclasses' );
+
+		if ( ! empty( $outline_styles ) ) {
+			$outline_styles = json_decode( html_entity_decode( $outline_styles, ENT_COMPAT ), true );
+			$input_styles   = $this->get_attribute( 'inputstyles' );
+
+			if ( ! empty( $input_styles ) ) {
+				$style_attrs = $input_styles;
+			}
+
+			$global_styles = wp_get_global_styles(
+				array( 'border' ),
+				array(
+					'block_name' => 'jetpack/input',
+					'transforms' => array( 'resolve-variables' ),
+				)
+			);
+
+			$legacy_border_size   = $outline_styles['border']['width'] ?? null;
+			$legacy_border_size   = is_numeric( $legacy_border_size ) ? $legacy_border_size . 'px' : $legacy_border_size;
+			$legacy_border_radius = $outline_styles['border']['radius'] ?? null;
+			$legacy_border_radius = is_numeric( $legacy_border_radius ) ? $legacy_border_radius . 'px' : $legacy_border_radius;
+
+			$border_size = $legacy_border_size ??
+				$outline_styles['border']['top']['width'] ??
+				$global_styles['width'] ??
+				$global_styles['top']['width'];
+
+			$border_radius = $legacy_border_radius ??
+				$global_styles['radius'];
+
+			$css_vars = $border_size ? '--jetpack--contact-form--border-size: ' . $border_size . ';' : '';
+			// Check if border radius is split or a single value.
+			if ( is_array( $border_radius ) ) {
+				// If corner radii are set on the top-left or bottom-left of the block, take the maximum of the two.
+				// We check the left side due to writing direction—this variable is used to offset text.
+				// TODO: this should factor in RTL languages.
+				$css_vars .= $border_radius ? '--jetpack--contact-form--border-radius: max(' . $border_radius['topLeft'] . ',' . $border_radius['bottomLeft'] . ');' : '';
+			} elseif ( isset( $border_radius ) ) {
+				$css_vars .= $border_radius ? '--jetpack--contact-form--border-radius: ' . $border_radius . ';' : '';
+			}
+
+			if ( 'outlined' === $form_style ) {
+				$css_vars .= '--jetpack--contact-form--notch-width: max(var(--jetpack--contact-form--input-padding-left, 16px), var(--jetpack--contact-form--border-radius));';
+			} elseif ( 'animated' === $form_style ) {
+				$legacy_border_left_size = $outline_styles['border']['width'] ?? null;
+				$legacy_border_left_size = is_numeric( $legacy_border_left_size ) ? $legacy_border_left_size . 'px' : $legacy_border_left_size;
+
+				$border_left_size = $legacy_border_left_size ??
+				$outline_styles['border']['left']['width'] ??
+				$global_styles['width'] ??
+				$global_styles['left']['width'];
+
+				$css_vars .= "--jetpack--contact-form--left-offset: calc(var(--jetpack--contact-form--input-padding-left, 16px) + {$border_left_size});";
+				$css_vars .= '--jetpack--contact-form--label-left: max(var(--jetpack--contact-form--left-offset), var(--jetpack--contact-form--border-radius));';
+				$css_vars .= "--jetpack--contact-form--field-padding: calc(var(--jetpack--contact-form--label-left) - {$border_left_size});";
```

Comment:
This PR simplifies these CSS vars https://github.com/Automattic/jetpack/pull/43469

I'm not convinced they're required for the animated style.

Reply:
@talldan 

Good news I think. I think we can remove the variable left padding for the Outlined style when there's lots of radius and border.

We've replicated the behaviour from trunk, but in trunk the indentation only occurs in the editor, and has no effect on the frontend in my testing.

Here's trunk:

| Trunk editor | Trunk frontend |
|--------|--------|
| <img width="400" alt="Screenshot 2025-05-16 at 11 07 08 am" src="https://github.com/user-attachments/assets/7b89d2c6-f68e-479d-a4b0-4c6744c2911f" /> | <img width="400" alt="Screenshot 2025-05-16 at 11 07 25 am" src="https://github.com/user-attachments/assets/8df3e23f-561d-4d39-8c3e-477471928457" /> |